### PR TITLE
[14.0][MIG]Web_timeline

### PR DESCRIPTION
**Pull request by Yoshiro009**

### CODE REVIEW

File: web_timeline/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Web timeline",
     "summary": "Interactive visualization chart to show events in time",
-    "version": "13.0.1.1.1",
+    "version": "14.0.1.1.1",
```
Comment (gdgellatly) [inline web_timeline/__manifest__.py]: `14.0.1.0.0`

Comment (newtratip) [reply web_timeline/__manifest__.py]: Fixed.

**Comment by dreispt** [comment]
This needs fixing. When testing https://github.com/OCA/project/pull/800 found:
```
Error:
TypeError: this.config.ControlPanel is undefined

http://bot.local:8078/web/static/src/js/views/abstract_view.js:201
Traceback:
init@http://bot.local:8078/web/static/src/js/views/abstract_view.js:201:38
OdooClass.extend/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:90:38
init@http://bot.local:8078/web_timeline/static/src/js/timeline_view.js:39:25
OdooClass.extend/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:90:38
Class@http://bot.local:8078/web/static/src/js/core/class.js:107:33
_createViewController@http://bot.local:8078/web/static/src/js/chrome/action_manager_act_window.js:191:24
newController@http://bot.local:8078/web/static/src/js/chrome/action_manager_act_window.js:534:18
_switchController@http://bot.local:8078/web/static/src/js/chrome/action_manager_act_window.js:568:29
_onSwitchView@http://bot.local:8078/web/static/src/js/chrome/action_manager_act_window.js:727:18
proxy/<@http://bot.local:8078/web/static/src/js/core/mixins.js:277:23
trigger@http://bot.local:8078/web/static/src/js/core/mixins.js:224:31
_trigger_up@http://bot.local:8078/web/static/src/js/core/mixins.js:339:34
_trigger_up@http://bot.local:8078/web/static/src/js/core/mixins.js:341:20
trigger_up@http://bot.local:8078/web/static/src/js/core/mixins.js:334:14
_addListener/<@http://bot.local:8078/web/static/src/js/owl_compatibility.js:484:43
__trigger@http://bot.local:8078/web/static/lib/owl/owl.js:4264:25
owl.Component.prototype.__trigger@http://bot.local:8078/web/static/src/js/component_extension.js:40:25
trigger@http://bot.local:8078/web/static/lib/owl/owl.js:4181:18
anonymous/p189.on.click@http://bot.local:8078/web/static/lib/owl/owl.js line 1695 > Function:21:107
invokeHandler@http://bot.local:8078/web/static/lib/owl/owl.js:625:21
handleEvent@http://bot.local:8078/web/static/lib/owl/owl.js:654:30
handler@http://bot.local:8078/web/static/lib/owl/owl.js:663:24
EventListener.handleEvent*updateEventListeners@http://bot.local:8078/web/static/lib/owl/owl.js:706:25
createElm@http://bot.local:8078/web/static/lib/owl/owl.js:905:34
createElm@http://bot.local:8078/web/static/lib/owl/owl.js:910:50
createElm@http://bot.local:8078/web/static/lib/owl/owl.js:910:50
createElm@http://bot.local:8078/web/static/lib/owl/owl.js:910:50
createElm@http://bot.local:8078/web/static/lib/owl/owl.js:910:50
patch@http://bot.local:8078/web/static/lib/owl/owl.js:1120:26
__patch@http://bot.local:8078/web/static/lib/owl/owl.js:4307:34
complete@http://bot.local:8078/web/static/lib/owl/owl.js:3583:35
flush/tasks<@http://bot.local:8078/web/static/lib/owl/owl.js:3344:40
flush@http://bot.local:8078/web/static/lib/owl/owl.js:3336:27
scheduleTasks/<@http://bot.local:8078/web/static/lib/owl/owl.js:3362:22
Async*scheduleTasks@http://bot.local:8078/web/static/lib/owl/owl.js:3361:18
start@http://bot.local:8078/web/static/lib/owl/owl.js:3292:18
addFiber/<@http://bot.local:8078/web/static/lib/owl/owl.js:3315:26
addFiber@http://bot.local:8078/web/static/lib/owl/owl.js:3301:20
mount@http://bot.local:8078/web/static/lib/owl/owl.js:4069:38
mount@http://bot.local:8078/web/static/src/js/owl_compatibility.js:519:26
start@http://bot.local:8078/web/static/src/js/views/abstract_controller.js:93:53
OdooClass.extend/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:90:38
start@http://bot.local:8078/web/static/src/js/views/basic/basic_controller.js:64:20
OdooClass.extend/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:90:38
_widgetRenderAndInsert/<@http://bot.local:8078/web/static/src/js/core/widget.js:440:25
promise callback*_widgetRenderAndInsert@http://bot.local:8078/web/static/src/js/core/widget.js:434:33
appendTo@http://bot.local:8078/web/static/src/js/core/widget.js:193:21
_startController@http://bot.local:8078/web/static/src/js/chrome/action_manager.js:854:34
_executeAction/<@http://bot.local:8078/web/static/src/js/chrome/action_manager.js:317:41
promise callback*_executeAction@http://bot.local:8078/web/static/src/js/chrome/action_manager.js:316:14
_executeWindowAction/</<@http://bot.local:8078/web/static/src/js/chrome/action_manager_act_window.js:313:33
promise callback*_executeWindowAction/<@http://bot.local:8078/web/static/src/js/chrome/action_manager_act_window.js:311:18
promise callback*_executeWindowAction@http://bot.local:8078/web/static/src/js/chrome/action_manager_act_window.js:256:53
_handleAction@http://bot.local:8078/web/static/src/js/chrome/action_manager_act_window.js:401:25
OdooClass.extend/Class.include/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:123:38
_handleAction@http://bot.local:8078/web/static/src/js/chrome/action_manager_report.js:164:28
OdooClass.extend/Class.include/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:123:38
_handleAction@http://bot.local:8078/account_reports/static/src/js/action_manager_account_report_dl.js:52:28
OdooClass.extend/Class.include/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:123:38
doAction/<@http://bot.local:8078/web/static/src/js/chrome/action_manager.js:165:25
promise callback*doAction@http://bot.local:8078/web/static/src/js/chrome/action_manager.js:157:50
loadState@http://bot.local:8078/web/static/src/js/chrome/action_manager_act_window.js:119:25
OdooClass.extend/Class.include/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:123:38
on_hashchange@http://bot.local:8078/web_enterprise/static/src/js/web_client.js:151:59
Async*show_application/<@http://bot.local:8078/web_enterprise/static/src/js/web_client.js:113:29
promise callback*show_application@http://bot.local:8078/web_enterprise/static/src/js/web_client.js:91:66
show_application@http://bot.local:8078/bus/static/src/js/web_client_bus.js:31:32
OdooClass.extend/Class.include/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:123:38
show_application@http://bot.local:8078/calendar/static/src/js/base_calendar.js:112:28
OdooClass.extend/Class.include/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:123:38
start/<@http://bot.local:8078/web/static/src/js/chrome/abstract_web_client.js:156:33
promise callback*start@http://bot.local:8078/web/static/src/js/chrome/abstract_web_client.js:154:16
start@http://bot.local:8078/web_enterprise/static/src/js/web_client.js:37:28
OdooClass.extend/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:90:38
start@http://bot.local:8078/bus/static/src/js/web_client_bus.js:23:24
OdooClass.extend/Class.include/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:123:38
start@http://bot.local:8078/web_enterprise/static/src/js/widgets/debug_manager.js:11:32
OdooClass.extend/Class.include/</prototype[name]</<@http://bot.local:8078/web/static/src/js/core/class.js:123:38
startWebClient@http://bot.local:8078/web/static/src/js/main.js:24:19
async*@http://bot.local:8078/web/static/src/js/main.js:26:5
processJob/def<@http://bot.local:8078/web/static/src/js/boot.js:260:43
processJob@http://bot.local:8078/web/static/src/js/boot.js:258:23
odoo.processJobs@http://bot.local:8078/web/static/src/js/boot.js:316:23
odoo.define@http://bot.local:8078/web/static/src/js/boot.js:154:14
@http://bot.local:8078/web/static/src/js/main.js:1:6
```

**Review by dreispt — CHANGES_REQUESTED** [review]
Functional tests, not working

**Comment by Murtaza-OSI** [comment]
Hello @dreispt
Please check the PR for web_timeline
https://github.com/OCA/web/pull/1851

**Comment by dreispt** [comment]
Superseded by https://github.com/OCA/web/pull/1851

**Pull request closed without merging**